src/renderer/src/components/ContextFiles/ContextFiles.tsx

```diff
@@ -352,6 +353,17 @@ export const ContextFiles = ({ baseDir, allFiles, showFileDialog }: Props) => {
                   <span
                     className={`select-none text-2xs overflow-hidden ${item.isFolder ? 'context-dimmed' : 'text-text-primary font-semibold'}`}
                     {...(item.isFolder ? { onClick: context.arrowProps.onClick } : {})}
+                    data-tooltip-id="context-files-tooltip"
+                    data-tooltip-content={(() => {
+                      if (!tokensInfo?.files || item.isFolder) {
+                        return '';
+                      }
+                      const fileTokenInfo = tokensInfo.files[item.index as string];
+                      if (!fileTokenInfo) {
+                        return '';
+                      }
+                      return `${t('usageDashboard.charts.tokens')}: ${fileTokenInfo.tokens || 0}, ${t('usageDashboard.charts.cost')}: $${(fileTokenInfo.cost || 0).toFixed(5)}`;
```

Comment:
It should be possible to return a React node here. Let's show the information so each info is on its own line.

Reply:
Actually, let's use the same notation we have in the Session info: "x tokens, $y.yyyyy"